### PR TITLE
feat(#4): Added create_asset functionality

### PR DESCRIPTION
**Pull request by suryansh0502**
Added create_asset functionality for following : 
- Web Asset
- Network Asset
- Cloud Asset
- Mobile Asset
- Asset with Multiple IPs

### CODE REVIEW

File: strobes_client/client.py [hunk 1]
```diff
@@ -21,40 +21,101 @@ def get_organization(self, org_id: str) -> resources.OrganizationResource:
...
+                f"{str(page)}"
+            )
         check_status_code(r)
         return resources.AssetListResource(r.json())
 
     def get_asset(self, org_id: str, asset_id: int) -> resources.AssetResource:
         r = self.s.get(
-            f"{self.app_url}api/v1/organizations/{org_id}/assets/"
-            f"{str(asset_id)}/")
+            f"{self.app_url}api/v1/organizations/{org_id}/assets/" f"{str(asset_id)}/"
```
Comment (Danansari) [inline strobes_client/client.py]: merge both format strings

File: strobes_client/client.py [hunk 2]
```diff
@@ -4,14 +4,14 @@
...
+
 
 class StrobesClient(BaseClient):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
-    def list_organizations(self, page: int = 1) -> \
-            resources.OrganizationListResource:
+    def list_organizations(self, page: int = 1) -> resources.OrganizationListResource:
         r = self.s.get(f"{self.app_url}api/v1/organizations/?page={str(page)}")
```
Comment (Danansari) [inline strobes_client/client.py:15]: add this in enums

File: strobes_client/client.py [hunk 3]
```diff
@@ -149,38 +228,41 @@ def update_vulnerability(self, org_id: str, asset_id: int,
...
         check_status_code(r)
         return resources.VulnerabilityResource(r.json())
 
-    def search_cve(self, org_id: str, search_term: str) \
-            -> resources.CVEListResource:
+    def search_cve(self, org_id: str, search_term: str) -> resources.CVEListResource:
         r = self.s.get(
-            f"{self.app_url}api/v1/organizations/{org_id}/cve/?q="
-            f"{search_term}")
+            f"{self.app_url}api/v1/organizations/{org_id}/cve/?q=" f"{search_term}"
```
Comment (Danansari) [inline strobes_client/client.py]: merge formatting string and pass it though enums

**Review by Danansari — CHANGES_REQUESTED** [review]
Please apply requested changes

**Review by Danansari — APPROVED** [review]

**Review by avinashbakshi01 — APPROVED** [review]

**Pull request merged**